src/pages/welcome/post/PostComponent.jsx

```diff
@@ -0,0 +1,63 @@
+import { getComments, getPost } from '../../../api/message.js';
+import { Heading, Layer, Section, Tile } from '@carbon/react';
+import { useEffect, useState } from 'react';
+
+const PostComponent = () => {
+  const [post, setPost] = useState();
+  const [comments, setComments] = useState([]);
+
+  const loadPost = async () => {
+    try {
+      const post = await getPost(1);
+      setPost(post);
+    } catch {
+      setPost('Failed to load message');
+    }
+  };
+
+  const loadComments = async () => {
+    try {
+      const comments = await getComments(1);
+      setComments(comments);
+    } catch {
+      setPost('Failed to load comments');
+    }
+  };
+
+  useEffect(() => {
+    loadPost();
+    loadComments();
+  }, []);
+
+  return (
+    <Section as="article" level={3}>
+      <Heading>Posts</Heading>
+      <Tile>
+        <Section as="article" level={3}>
+          <Section level={4}>
+            <Heading>{post?.title ?? 'Loading...'}</Heading>
+            <div>{post?.body}</div>
+          </Section>
+        </Section>
+
+        <Section as="article" level={3}>
+          <Heading>Comments</Heading>
+          <Section as="article" level={4}>
+            <Layer level={1}>
+              <Tile>
```

Comment:
Did we really mean tile inside tile? Are we using this to gain extra padding around the comments map?